addons/knowledge/static/src/webclient/commands/knowledge_providers.js

```diff
@@ -0,0 +1,148 @@
+/** @odoo-module */
+
+
+import { _lt } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { HotkeyCommandItem } from "@web/core/commands/default_providers";
+
+const { Component } = owl;
+const { xml } = owl.tags;
+
+
+class KnowledgeCommand extends Component {}
```

Comment:
You still miss the docstring on those classes.

Reply:
I don't really see how docstring would make sense here... this is just the regular way to extend components:
see `/addons/web/static/src/webclient/menus/menu_providers.js`, `/addons/mail/static/src/webclient/commands/mail_providers.js`

What about simple comments like:
```js
// Articles command
class KnowledgeCommand extends Component {}
...
// "Not found, create one" command
class Knowledge404Command extends Component {}
...
// Advanced search command
class KnowledgeExtraCommand extends HotkeyCommandItem {}
```